server/graphql/v2/input/ConnectedAccountCreateInput.ts

```diff
@@ -0,0 +1,22 @@
+import { GraphQLString, GraphQLInputObjectType } from 'graphql';
+import GraphQLJSON from 'graphql-type-json';
+import { ConnectedAccountService } from '../enum/ConnectedAccountService';
+
+/**
+ * An input for ConnectedAccount that can be used for either editing or creating.
+ */
+const ConnectedAccountCreateInput = new GraphQLInputObjectType({
+  name: 'ConnectedAccountCreateInput',
+  fields: {
+    clientId: { type: GraphQLString },
+    data: { type: GraphQLJSON },
+    id: { type: GraphQLString },
```

Comment:
what's id?

Reply:
The unique id of the connected account, although it is not needed since we're just allowing the creation of new ConnectedAccounts.
I'm removing this.